src/kfactory/decorators.py

```diff
@@ -35,6 +36,7 @@
 
 if TYPE_CHECKING:
     from collections.abc import Callable, Iterable, Sequence
+    from pathlib import Path
 
     from cachetools.keys import _HashedTuple  # type: ignore[attr-defined,unused-ignore]
```

Comment:
```suggestion
    _HashedTuple = tuple[Any, ...]
```
Now that we use this in `@cell`, pylance complains at says the function type is unknown. Because we dont really need to use the `_HashedTuple`, this is a lot nicer to me

Reply:
I think the problem of pylance is KCellFunc actually. We should replace that one with `Callable[KCellParams, KC]`

Reply:
its that it sees _HashedTuple as unknown, well thats one problem at least

Reply:
Got it. We can just replace it with `int`, because I believe in the end it will be mapped to `int`